contracts/core/RebalancingSetTokenFactory.sol

```diff
@@ -34,32 +34,45 @@ contract RebalancingSetTokenFactory {
 
     /* ============ State Variables ============ */
 
-    // Address of the Core contract
+    // Address of the Core contract used to verify factory when creating a Set
     address public core;
 
+    // Minimum amount of time between rebalances in seconds
+    uint256 public minimumRebalanceInterval;
+
+    // Minimum amount of time users can review proposals
+    uint256 public minimumProposalPeriod;
+
     // ============ Structs ============
 
     struct InitRebalancingParameters {
         address manager;
-        uint256 proposalPeriod;
         uint256 rebalanceInterval;
+        uint256 proposalPeriod;
         uint256 entranceFee;
         uint256 rebalanceFee;
     }
 
     /* ============ Constructor ============ */
 
     /**
-     * Set core
+     * Set core. Also requires a minimum rebalance interval and minimum proposal periods that are enforced
+     * on RebalancingSetToken
      *
-     * @param  _core   The address of deployed core contract
+     * @param  _core                       Minimum amount of time users can review proposals
+     * @param  _minimumRebalanceInterval   Minimum amount of time between rebalances in seconds
+     * @param  _minimumProposalPeriod      Minimum amount of time users can review proposals
```

Comment:
Nit: specify seconds here too since it's specified for `_minimumRebalanceInterval`

Reply:
+1